test/evergreen.yml

```diff
@@ -4107,6 +4107,46 @@ buildvariants:
     - name: ".stress-test-zseries-2"
     - name: ".stress-test-zseries-3"
 
+- name: ubuntu2004-arm64
+  display_name: "~ Ubuntu 20.04 ARM64"
+  run_on:
+  - ubuntu2004-arm64
+  expansions:
+    test_env_vars:
+      WT_TOPDIR=$(git rev-parse --show-toplevel)
+      WT_BUILDDIR=$WT_TOPDIR/cmake_build
+      LD_LIBRARY_PATH=$WT_BUILDDIR
+    posix_configure_flags:
+      -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/mongodbtoolchain_v3_gcc.cmake
+      -DCMAKE_C_FLAGS="-ggdb"
+      -DHAVE_DIAGNOSTIC=1
+      -DENABLE_PYTHON=1
+      -DENABLE_ZLIB=1
+      -DENABLE_SNAPPY=1
+      -DENABLE_STRICT=1
+      -DCMAKE_INSTALL_PREFIX=$(pwd)/LOCAL_INSTALL
+    python_binary: '/opt/mongodbtoolchain/v3/bin/python3'
+    smp_command: -j $(echo "`grep -c ^processor /proc/cpuinfo` * 2" | bc)
+    cmake_generator: Ninja
+    make_command: ninja
+  tasks:
+    - name: compile
+    - name: make-check-test
+    - name: unit-test
+    - name: fops
+    - name: linux-directio
+    - name: checkpoint-filetypes-test
+    - name: unit-test-zstd
+    - name: unit-test-long
+    - name: spinlock-gcc-test
+    - name: spinlock-pthread-adaptive-test
+    - name: compile-wtperf
+    - name: wtperf-test
+    - name: ftruncate-test
+    - name: long-test
+    - name: configure-combinations
+    - name: format-smoke-test
```

Comment:
Do we really hope to run a "full set" when first-time introducing this new ARM64 builder? For Windows and macOS builders we run only `make-check-test`, `unit-test`, and `fops`.

Reply:
The original ticket was mostly requesting testing on par with our other architectures (this test set was mostly based on what we are doing with our RHEL variant). I figured this was a comprehensive set of tests to satisfy that requirement, as there are some unique performance differences and characteristics between x86_64 and arm64 hosts (where our macOS and Windows variants are mostly just more x86 tests)

Reply:
I'm happy to remove some of these specific tests, I'm concerned though that running just `make-check-test`, `unit-test`, and `fops` is a bit limited in testing scope.

Reply:
Alright, sounds good to me to keep the current scope. Just rerun/restart the patch build a few times if possible, hoping to rule out potential intermittent failures.

Reply:
Yep, so I've re-ran the variant a couple of times now and its been pretty green:
- https://spruce.mongodb.com/version/61fb57660ae60650cae29538/tasks
- https://spruce.mongodb.com/version/61fc72380305b973ad2c544f/tasks?page=0&variant=%5Eubuntu2004-arm64%24
- https://spruce.mongodb.com/version/61fc83292a60ed21514a1815/tasks
- https://spruce.mongodb.com/version/61fc8dfa0ae60670db964891/tasks
Will go ahead and merge now.